steely/plugins/url_expander.py

```diff
@@ -0,0 +1,34 @@
+from urllib.request import Request, urlopen
+import json
+
+
+def expand(url):
+    API_ADDRESS = 'http://x.datasig.io/short?url={}'
```

Comment:
I don't see the problem with having it local. I prefer keeping things as local as possible until it's necessary to lift them to a higher scope.